kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/listeners/invocation.kt

```diff
@@ -1,9 +1,9 @@
 package io.kotest.core.listeners
 
+import io.kotest.core.extensions.Extension
 import io.kotest.core.test.TestCase
 
-@Suppress("DEPRECATION") // Remove when removing Listener
-interface BeforeInvocationListener : Listener {
+interface BeforeInvocationListener : Extension {
```

Comment:
For more consistency, should we also rename all `*Listener` interfaces into `*Extension`? Such as `BeforeInvocationListener => BeforeInvocationExtension`

Reply:
Imo the name doesn't really matter

Reply:
If starting from zero I'd agree